src/olympia/amo/management/commands/check_locales_completion_rate.py

```diff
@@ -0,0 +1,175 @@
+                )
+                locales_below_threshold.add(locale)
+        # Somewhat inefficient but good enough check to remove any locales that
+        # both below and above threshold because of differences between the
+        # projects.
+        for locale in locales_above_threshold:
+            if locale in locales_below_threshold:
+                self.stdout.write(f'❌ {locale} is in both sets !')
+                locales_above_threshold.remove(locale)
+        return locales_below_threshold, locales_above_threshold
+
+    def fetch_data(self, project):
+        q = re.sub(r'\s', '', self.QUERY_TEMPLATE % {'project': project})
+        self.stdout.write(f'Calling pontoon with {q}')
+        response = requests.get(self.PONTOON_API, {'query': q})
+        response.raise_for_status()
+        data = response.json().get('data', {})
+        return data
+
+    def get_completion(self, locale_data):
+        return round(locale_data['approvedStrings'] / locale_data['totalStrings'] * 100)
+
+    def get_locale(self, locale_data):
+        return locale_data['locale']['code']
+
+    def pretty_locale_name(self, locale):
+        pretty = ALL_LANGUAGES.get(locale, {}).get('english')
+        if pretty:
+            return f'{pretty} [{locale}]'
+        return locale
+
+    def warn_about(self, *, locales_below_threshold, locales_above_threshold):
+        locales_to_keep_despite_being_below_threshold = set()
+        for locale in self.LOCALES_TO_ALWAYS_KEEP:
+            if locale in locales_below_threshold:
+                self.stdout.write(f'⚠️ {locale} should be kept but is below threshold !')
+                locales_below_threshold.remove(locale)
+                locales_to_keep_despite_being_below_threshold.add(locale)
+        context = {
+            'COMPLETION_THRESHOLD': self.COMPLETION_THRESHOLD,
+            'locales_above_threshold': '\n- '.join(
+                sorted(
+                    [
+                        self.pretty_locale_name(locale)
+                        for locale in locales_above_threshold
+                    ]
+                )
+            ),
+            'locales_below_threshold': '\n- '.join(
+                sorted(
+                    [
+                        self.pretty_locale_name(locale)
+                        for locale in locales_below_threshold
+                    ]
+                )
+            ),
+            'locales_to_keep_despite_being_below_threshold': '\n- '.join(
+                sorted(
+                    [
+                        self.pretty_locale_name(locale)
+                        for locale in locales_to_keep_despite_being_below_threshold
+                    ]
+                )
+            ),
+        }
+        template = loader.get_template(self.EMAIL_TEMPLATE_PATH)
+        message = template.render(context)
+        send_mail(self.EMAIL_SUBJECT, message, recipient_list=self.EMAIL_RECIPIENTS)
```

Comment:
AFAICT the `dry_run` is not used here.. it probably should be.

Reply:
Also, could skip if both sets are empty to prevent needless email sending.

Reply:
Ah yeah forgot the `dry_run`... I don't think it's needed in the end: we'll never do a dry run in prod, and locally we already have `FakeEmail`